tests/kernel/install_ltp.pm

```diff
@@ -258,7 +263,7 @@ sub setup_network {
     # boo#1017616: missing link to ping6 in iputils >= s20150815
     assert_script_run('which ping6 >/dev/null 2>&1 || ln -s `which ping` /usr/local/bin/ping6');
 
-    unless (is_transactional) {
+    unless (is_transactional || has_kea) {
```

Comment:
We have now confirmation that dhcp-server is not in SLE16. I would just do  `unless (is_transactional || is_sle('16+')) {`. kea needs seperate action and proper test adapation and I don't want to proceed as is above suggested.

Reply:
Works for me